documentation/src/main/wiki/BasicSetup.md

```diff
@@ -5,104 +5,65 @@ This setup makes use of the Vagrant module, a pseudo-cookbook which is responsib
 #### Overview
 
 Currently a virtual machine with the following services is created:
-
 * Karaf
 * Apache
 * Tomcat
 * ActiveMQ
 * MongoDb
-* Brosermob
-* Firefox
-* X environment
+* Browsermob
+* Selenium Grid hub
 
-#### AET Services
+![aet-setup-vagrant](assets/diagrams/aet-setup-with-vagrant.png)
 
 All services run using default ports. For communication please use the following IP address:
-
 * `192.168.123.100`
 
-#### General Prerequisites
+#### Prerequisites
 
-By default the Vagrant virtual machine needs 3 GB of RAM and 2 vCPUs, so please make sure that you have enough memory on your machine (8 GB is minimum, 16 GB is recommended though).
+By default the Vagrant virtual machine needs 3 GB of RAM and 2 vCPUs, so please make sure that 
```

Comment:
Shouldn't be `8 GB` here?

Reply:
Actually it's 6 GB and 4 vCPUs: 
https://github.com/Cognifide/aet/blob/master/vagrant/Vagrantfile#L58-L59
And it hasn't been changed since the first commit in this repo.